container/sha256.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A wrapper to have a portable SHA-256 tool."""
```

Comment:
This is just a duplicate of https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/hash/sha256.py, right? I do hope we could reuse that instead.

Reply:
It is, the fact that the tool didn't need to come over didn't dawn on me here. Dropping it for a direct reference to the bazel_tools target.